### ISSUE TITLE
Seemingly impossible time steps in archival tag data

### ISSUE THREAD

**Issue by camrinbraun** [camrinbraun/tags2etuff#29]
from @marosteg:

_The swordfish archival tag issue (for PTT 110496) was the occasional dawn and/or dusk spikes in depth/light during crepuscular descent/ascent, which were reported at intervals inconsistent with the rest of the archive. For example, if the archive is at 5-s intervals, the erroneous values might show up in between (e.g., 12:30:30, 12:30:33, 12:30:35). The change in depth in these erroneous entries were clearly impossible given the speeds required to achieve them (e.g., 100+ m in 2 s)._

The question then is whether or not this is an issue with `tag_to_etuff()` or something that is inherent to the tag dataset (and needs to be filtered out).

**Comment by camrinbraun** [comment]
I see the issue with the timesteps in this dataset _but it does not occur in the raw -Archive.csv data_ suggesting it is related to the eTUFF conversion. However, I've tried several different iterations of converting that data to eTUFF and cannot re-create the issue (i.e. all resulting eTUFF files are correct, without time issue). Re-try your analysis with the `_eTUFF.txt` file for this tag that I uploaded to MPG -> archival_martini. I just wrote over the previous eTUFF file for this fish. Let me know if, for whatever reason, this issue still persists.
